addon/src/main/java/org/vaadin/risto/stepper/AbstractStepper.java

```diff
@@ -259,4 +281,12 @@ protected abstract T parseStringValue(String value)
      * @return the type of the value in this field
      */
     protected abstract Class<T> getValueType();
+
+    @FunctionalInterface
+    public interface ClickListener extends Serializable {
+        Method BUTTON_CLICK_METHOD = ReflectTools.findMethod(ClickListener.class,
```

Comment:
I would prefer a separate classes from Button. Even if they would be mostly copy-paste.

Reply:
done